stream-chat-android-offline/src/main/java/io/getstream/chat/android/livedata/entity/MessageEntity.kt

```diff
@@ -12,6 +12,12 @@ internal data class MessageEntity(
     @Embedded val messageInnerEntity: MessageInnerEntity,
     @Relation(entity = AttachmentEntity::class, parentColumn = "id", entityColumn = "messageId")
     val attachments: List<AttachmentEntity>,
+    /** the reactions from the current user */
+    @Relation(entity = ReactionEntity::class, parentColumn = "id", entityColumn = "messageId")
+    val ownReactions: List<ReactionEntity> = emptyList(),
+    /** the last 5 reactions on this message */
```

Comment:
It's just copy-paste but I think that `latestReactions` can contain more than 5 reactions

Reply:
I'm not sure about logic of reaction domain. We can remove this if it's not true. Just as you mentioned I copied it

Reply:
@bychkovdmitry can you confirm that?